style.css

```diff
@@ -409,7 +421,7 @@ section#search-modal {
 
 .features-container .wsu-home-headline-wrapper-open {
 	display: block;
-	-webkit-animation: fadeIn 1.5s;
+	-webkit-animation: fadein 1.5s;
```

Comment:
Remove the -webkit-animation. I believe all of the -webkit browsers now honor animation